src/sql/workbench/api/node/mainThreadConnectionManagement.ts

```diff
@@ -43,6 +43,19 @@ export class MainThreadConnectionManagement implements MainThreadConnectionManag
 		this._toDispose = dispose(this._toDispose);
 	}
 
+	public $getConnections(activeConnectionsOnly?: boolean): Thenable<azdata.connection.ConnectionProfile[]> {
```

Comment:
>$getConnections [](start = 8, length = 15)

Please add tests for this - this is a really important API and we want to make sure it's returning connections as expected